src/components/Modals/Modal.vue

```diff
@@ -11,12 +11,16 @@
         :isOnboard="isOnboard"
```

Comment:
lowkey f in the chat for whoever made factored this modal out. it made my life very hard (communicating from parent to child and from child to parent). F.

Reply:
Yeah Modal.vue has kind of been used in very stange/bad ways since it was created. Would be a good idea to restructure it sometime 😕

Reply:
for suree lol